.github/workflows/upload_release_assets.yml

```diff
@@ -160,3 +160,30 @@ jobs:
         asset_path: ./assets/antrea-windows.yml
         asset_name: antrea-windows.yml
         asset_content_type: application/octet-stream
+    - name: Upload antrea-agent-windows-x86_64.exe
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: ./assets/antrea-agent.exe
+          asset_name: antrea-agent-windows-x86_64.exe
+          asset_content_type: application/octet-stream
+    - name: Upload antrea-cni-windows-x86_64.exe
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: ./assets/antrea-cni.exe
+          asset_name: antrea-cni-windows-x86_64.exe
+          asset_content_type: application/octet-stream
```

Comment:
don't we need to build these in `prepare-assets.sh`? We currently only build the antctl binaries there.

Reply:
Hi @antoninbas , I also have added the build steps for these files in `prepare-assets.sh`.
Could you help review the `hack/release/prepare-assets.sh` ?

Reply:
My bad, I missed this line: `BINDIR="$OUTPUT_DIR" make windows-bin`. LGTM